corehq/apps/app_manager/views/modules.py

```diff
@@ -1009,36 +1009,47 @@ def _update_search_properties(module, search_properties, lang='en'):
     True
 
     """
+
+    # Replace translation for current language in a translations dict
+    def _update_translation(old_obj, new_data, old_attr, new_attr=None):
+        new_attr = new_attr or old_attr
```

Comment:
this is kind of confusing.  If I've got it right, it's:
* `old_attr`: the attr in `current`
* `new_attr`: the attr in `prop`?

I'll think about whether I can come up with anything more obvious, but nothing comes to mind offhand.

Reply:
yep, that's it, open to suggestions on the naming (and structure - I don't think this code is especially elegant, it's just good enough)